packages/react/src/themes/teams-high-contrast/components/Toolbar/toolbarVariables.ts

```diff
@@ -0,0 +1,16 @@
+import { ToolbarVariables } from '../../../teams/components/Toolbar/toolbarVariables'
+
+export default (siteVars: any): Partial<ToolbarVariables> => ({
+  foreground: siteVars.colors.white,
```

Comment:
just the thought unrelated to this PR - really think that we need to introduce `siteVariables`'s types for Teams theme - otherwise it might become to be pretty complicated story to maintain all this, especially given that name of the variable is not fixed on the client side